src/lib/time-picker/time-picker-foundation.ts

```diff
@@ -588,10 +592,14 @@ export class TimePickerFoundation implements ITimePickerFoundation {
   }
 
   private _findClosestOptionIndex(value: number, options: Array<IListDropdownOption<ITimePickerOptionValue>>): number {
-    const closestItem = options.reduce((prev, curr) => {
+    if (options.length) {
```

Comment:
This seems uneccessary as the two places it's called from should prevent this method from being executed if options.length is 0

Reply:
fixed with most recent push